webpack.config.ts

```diff
@@ -42,6 +42,8 @@ export default {
     rules: [
       { test: /\.ts/, loader: 'ts-loader' },
       { test: /\.glsl|css/, loader: 'raw-loader' },
+      { test: /\.gltf/, loader: 'gltf-webpack-loader' },
+      { test: /\.bin|png|svg|jpg|gif/, loader: 'file-loader' },
```

Comment:
Found two working ways to load gltf:
1. through file-loader, but you need to import supplemental files  (bin, png ...) somewhere to GLTFLoader to find them.
2. through special loader, which will load supplemental files automatically.
Chose the second way as there can be a lot of supplemental files and importing them all can be dispiriting.